RELEASE_NOTES.md

```diff
@@ -1,6 +1,7 @@
 # Release Notes
 
 ## 6.1.0 - 2024-07-27
+* BUGFIX: Allow RunSettingsArguments to be set, thanks @TheAngryByrd - https://github.com/fsprojects/FAKE/pull/2771
```

Comment:
6.1.0 has been released 🙂
```suggestion

## 6.1.1 - 2024-08-07
* BUGFIX: Allow RunSettingsArguments to be set, thanks @TheAngryByrd - https://github.com/fsprojects/FAKE/pull/2771

## 6.1.0 - 2024-07-27
```

Reply:
Hmm we need to make sure we have Releases lined up with actual releases then: https://github.com/fsprojects/FAKE/releases

Reply:
#2800